Tests/Svg.UnitTests/XxeVulnerabilityTests.cs

```diff
@@ -39,6 +41,35 @@ public void XxeResolveExternalResources(bool resolveExternalResources)
                 DeleteSecretFile(secretFilePath);
             }
         }
+        
+        [TestCase(ExternalType.None, "__pull_request-873-01", "__pull_request-873-01-ExternalReferencesDisabled")]
```

Comment:
Include this test in `ImageComparisonTest`.

https://github.com/svg-net/SVG/pull/873#discussion_r632527485

Reply:
There is no need for this, as the image comparison tests are already executed here.

Reply:
I commented that DOM test should be done here because images comparison test is uncertain...

Reply:
If it looks like it will be in time for this release, I will create a PR.

Reply:
Go ahead 👍